main-app/app/scripts/gameboard/gameboard-services/gameboard-service.js

```diff
@@ -0,0 +1,41 @@
+angular.module('Server.MyModule')
+    .service('FreshGame', ['PlayerService','NewGame','CoreData','ServerTurn','$state',  function (playerService, newGame, coreData, serverTurn, $state) {
+        var outcome = 'Continue',
+            me = this,
+            gameEnded = function () {
+                if (outcome === "Win") {
+                    $state.go('route4');
+                }
+                else if (outcome === "Draw") {
+                    $state.go('route5');
+                }
+            },
+            updateInformaton = function (response) {
+                coreData.gameBoard = response.gameboard;
+                coreData.currentGameState = response.outcome;
+                outcome = response.outcome;
+                coreData.winner = response.winner;
+            };
+
+        me.startGame = function () {
+            me.Player1 = playerService.player1;
+            me.Player2 = playerService.player2;
+            newGame.newGame(me.Player1, me.Player2)
```

Comment:
newGame.newGame is badly named and indicative of a problem - newGame and makeMove functions should be in the same proxy class - give me a shout if you need help.